src/leader_scheduler.rs

```diff
@@ -610,6 +615,17 @@ pub mod tests {
     fn test_num_ticks_left_in_slot() {
         let leader_scheduler = LeaderScheduler::new(&LeaderSchedulerConfig::new(10, 2, 1));
 
+        assert_eq!(leader_scheduler.num_ticks_left_in_block(0, 0), 9);
```

Comment:
Wait, move this to a new `test_num_ticks_left_in_block()` function please.  No need to clutter `test_num_ticks_left_in_slot()` with stuff unrelated to `num_ticks_left_in_slot` (ie, the thing this test is testing)

Reply:
will do

Reply:
it got this way because I deprecated the _in_slot() version, initially...